app/assets/stylesheets/modern-business.scss

```diff
@@ -131,9 +168,8 @@ header.carousel .fill#slider-3 {
   color: #000000
 }
 
-
-#logo {
-  font-family: 'Quattrocento', serif;
+.front-button {
+  background-color: #001B4D;
```

Comment:
Color literals like `#001B4D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#001B4D` should be written as `#001b4d`